### ISSUE TITLE
[7.0.0] Using JsonVariantConst causes "no matching function" error

### ISSUE THREAD

**Issue by BlueAndi** [bblanchon/ArduinoJson#2019]
Example code for reproduction:
```cpp
    JsonDocument    jsonDoc;
    JsonObject      jsonAbc = jsonDoc["abc"].add<JsonObject>();

    jsonAbc["a"] = "a";
    jsonAbc["b"] = "b";
    jsonAbc["c"] = "c";

    {
        JsonObjectConst     value = jsonDoc["abc"];
        JsonVariantConst    jsonA = value["a"]; /* ArduinoJson/Object/JsonObjectConst.hpp:105:38: error: no matching function for call to 'ArduinoJson::V700HB4::detail::ObjectData::getMember(const ArduinoJson::V700HB4::detail::ObjectData* const&, ArduinoJson::V700HB4::detail::StringAdapter<const char*, void>::AdaptedString)' */
    }
```

**Comment by bblanchon** [comment]
Hi @BlueAndi,

Thank you very much for reporting this bug :heart:
I'll publish the fix in the next few days.

Best regards,
Benoit

**Comment by BlueAndi** [comment]
Hi @bblanchon, you are welcome. Happy new year Benoit and thanks for the great work and your effort!

**Comment by bblanchon** [comment]
The fix was published with [ArduinoJson 7.0.1](https://github.com/bblanchon/ArduinoJson/releases/tag/v7.0.1).
